src/py/flwr/server/strategy/dp_fixed_clipping.py

```diff
@@ -185,3 +188,150 @@ def evaluate(
     ) -> Optional[Tuple[float, Dict[str, Scalar]]]:
         """Evaluate model parameters using an evaluation function from the strategy."""
         return self.strategy.evaluate(server_round, parameters)
+
+
+class DifferentialPrivacyClientSideFixedClipping(Strategy):
+    """Wrapper for Central DP with Client Side Fixed Clipping.
+
+    Use fixedclipping_mod modifier at the client side.
```

Comment:
Putting code references in backticks makes it easier for the user:
```suggestion
    Use `fixedclipping_mod` modifier at the client side.
```